hosted/json-schemas/offering.schema.json

```diff
@@ -142,12 +142,31 @@
     "requiredClaims": {
       "type": "object",
       "description": "PresentationDefinition that describes the credential(s) the PFI requires in order to provide a quote."
+    },
+    "cancellation": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether cancellation is enabled for this offering"
+        },
+        "terms_url": {
+          "type": "string",
+          "description": "A link to a page that describes the terms of cancellation"
+        },
+        "terms": {
+          "type": "string",
+          "description": "A human-readable description of the terms of cancellation in plaintext"
+        }
+      },
+      "required": ["enabled"]
```

Comment:
Did we check with legal to see if cancelation terms are a requirement? According to Chad, it is

> Companies are required to provide the terms of cancellation under the EFTA (Electronic Fund Transfer Act) regulation. Specifically, the regulation mandates that remittance transfer providers must disclose certain key information to consumers, including the terms and conditions of cancellation rights.
> 
> The terms of cancellation must include:
> 
> * The right to cancel: Informing the consumer of their right to cancel the remittance transfer within 30 minutes.
> * Instructions on how to cancel: Providing clear and understandable instructions on how to execute the cancellation.
> * Conditions for cancellation: Stating that the right to cancel is applicable only if the transfer has not been picked up by the recipient or deposited into the recipient’s account.
>
> This information is usually provided in the pre-payment disclosures and the receipt given to the consumer at the time of the transaction. This ensures that consumers are fully aware of their rights and the procedures for exercising them.

Reply:
ok, so we have either `terms` or `terms_url` - is what you're asking for to include either of those (or both) in a cancellation detail object?

Reply:
created an issue for this. https://github.com/TBD54566975/tbdex/issues/349